warehouse/postgres/postgres_test.go

```diff
@@ -51,6 +53,13 @@ func (*TestHandle) VerifyConnection() error {
 }
 
 func TestPostgresIntegration(t *testing.T) {
+	// Setting up the test configuration
+	require.NoError(t, testhelper.SetConfig([]warehouseutils.KeyValue{
+		{
```

Comment:
Since it is a warehouse-level config, let's not test it in a destination-specific config because we can't have similar functionality for other destinations. Instead, let's move this into a separate suite such as warehouse_integration.test for this.

Reply:
Because same enable/disable can't be set for other destination as well.

Reply:
Then it's fine.